src/HaskellerAnswers.hs

```diff
@@ -27,14 +30,20 @@ runHaskellerAnswers = do
         }
 
 data Model = Model
-    { modelAllAnswers    :: !(NonEmpty Text)
+    { modelAllAnswers    :: ![Text]
```

Comment:
Maybe we should rename it to `modelRemainingAnswers` or something which is more suitable by meaning?